src/backend/tests/unit/components/tools/test_arxiv_component.py

```diff
@@ -0,0 +1,138 @@
+from unittest.mock import patch
+
+import pytest
+
+from tests.base import ComponentTestBaseWithClient
+
+
+class TestArXivComponent(ComponentTestBaseWithClient):
+    def test_all_versions_have_a_file_name_defined(self, file_names_mapping):
+        """Test if all versions have a file name defined."""
+        assert len(file_names_mapping) > 0
+        for mapping in file_names_mapping:
+            assert "version" in mapping
+            assert "module" in mapping
+            assert "file_name" in mapping
+
+    def test_component_versions(self, default_kwargs, file_names_mapping):
+        """Test component compatibility across versions."""
+        from src.backend.base.langflow.components.tools.arxiv import ArXivComponent
+
+        # Test current version
+        component = ArXivComponent(**default_kwargs)
+        frontend_node = component.to_frontend_node()
+        assert frontend_node is not None
+
+        # Test backward compatibility
+        for mapping in file_names_mapping:
+            try:
+                module = __import__(
+                    f"src.backend.base.langflow.components.{mapping['module']}",
+                    fromlist=[mapping["file_name"]],
+                )
+                component_class = getattr(module, mapping["file_name"])
+                component = component_class(**default_kwargs)
+                frontend_node = component.to_frontend_node()
+                assert frontend_node is not None
+            except (ImportError, AttributeError) as e:
+                pytest.fail(f"Failed to load component version {mapping['version']}: {e!s}")
+
+    @pytest.fixture
+    def component_class(self):
+        from src.backend.base.langflow.components.tools.arxiv import ArXivComponent
+
+        return ArXivComponent
+
+    @pytest.fixture
+    def default_kwargs(self):
+        return {
+            "search_query": "quantum computing",
+            "search_type": "all",
+            "max_results": 10,
+            "_session_id": "test-session",
+        }
+
+    @pytest.fixture
+    def file_names_mapping(self):
+        return [
+            {
+                "version": "1.0.0",  # Versão inicial do componente
+                "module": "tools.arxiv",
+                "file_name": "ArXivComponent",
+            }
+        ]
```

Comment:
New components shouldn't have file_names_mapping and please avoid portuguese in the code.
```suggestion
        return []
```